.github/workflows/pr-assessment.yaml

```
@@ -0,0 +1,14 @@
+name: GitHub PR Size Assessment
+on:
+  pull_request:
+    types:
+      - closed
+
+jobs:
+  assess-pr-size-on-merge:
+      uses: rainlanguage/github-chore/.github/workflows/pr-assessment.yml@main
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 4 but found 6

(indentation)